lib/Cake/Network/CakeSocket.php

```diff
@@ -44,7 +44,8 @@ class CakeSocket {
 		'host' => 'localhost',
 		'protocol' => 'tcp',
 		'port' => 80,
-		'timeout' => 30
+		'timeout' => 30,
+		'cryptoType' => 'tls'
```

Comment:
What other values would go in here? There aren't any tests or documentation for this config option.